crates/holochain_integrity_types/src/capability/secret.rs

```diff
@@ -15,7 +15,7 @@ pub type CapSecretBytes = [u8; CAP_SECRET_BYTES];
 ///
 /// @todo enforce that secrets are unique across all grants in a chain.
 #[derive(Clone, Copy, SerializedBytes)]
-pub struct CapSecret(CapSecretBytes);
+pub struct CapSecret(pub CapSecretBytes);
```

Comment:
I know it's a bit of a pain, but I don't think we want to encourage pub access to the bytes here... you can use the arbitrary impl below for your tests, or add a random constructor guarded by a `#[cfg(test)]`.